single_tech_samples/synapseanalytics/sample2_synapseserverless/synapseartifacts/notebooks/Nb_NYCTaxi_Config_Operations_Library_No_Wheel.ipynb

```diff
@@ -0,0 +1,431 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "## Install Required Python Modules\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "nteract": {
+          "transient": {
+            "deleting": false
+          }
+        }
+      },
+      "source": [
+        "*This module installation can be moved to package installation using \"requirements.txt\"*\n",
```

Comment:
as above, do we still need the version with the pip install command given we have the wheel version now?

Reply:
This is included in the No_Wheel version - so yes.